src/components/v5/common/MemberSignature/MemberSignature.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+
+import { useIntl } from 'react-intl';
+import UserAvatar from '~v5/shared/UserAvatar';
+import { MemberSignatureProps } from './types';
+
+const displayName = 'v5.common.MemberSignature';
+
+const MemberSignature: FC<MemberSignatureProps> = ({ user, isSigned }) => {
```

Comment:
Please replace `isSigned` with something more suitable

Reply:
Changed to isChecked

Reply:
I meat that isSigned sounds like this person is signed and not that he or she signed something